flocker/control/_protocol.py

```diff
@@ -709,6 +714,28 @@ def disconnected(self, connection):
         """
         self.connections.remove(connection)
 
+    def _execute_broadcast_update(self):
+        """
+        Actually executes a broadcast update to all current connections.
+        """
+        self._broadcast_scheduled = False
+        self._send_state_to_connections(self.connections)
+
+    def _schedule_broadcast_update(self):
+        """
+        Ensure that there is a pending broadcast update call.
+
+        This is called when the state or configuration is updated, to trigger
+        a broadcast of the current state and configuration to all nodes.
+
+        In general, it only schedules an update to be broadcast 1 second later
+        so that if we recieve multiple updates within that second they are
+        coalesced down to a single update.
+        """
+        if not self._broadcast_scheduled:
+            self._broadcast_scheduled = True
+            self._reactor.callLater(1.0, self._execute_broadcast_update)
```

Comment:
Used a slightly different name for the constant. Added storing the callLater, and a test to verify that we cancel all callLaters once stopService has been called.